requirements-dev.txt

```diff
@@ -32,7 +32,7 @@ pynvml == 8.0.4
 numpy == 1.22.0
 
 # For layerwise gradient scaler
-sklearn >= 0.0
+scikit-learn >= 0.0
```

Comment:
nice! please use == to ensure it is not changing in the future when a new release is made by scikit-learn.